### PR TITLE
feat: added pre-wired rule for IS -> COS

### PR DESCRIPTION
**Pull request by Ak-sky**
### Description

Added pre-wired rule for IS -> COS

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- When merging, use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

File: examples/fscloud/variables.tf [hunk 1]
```diff
@@ -69,3 +69,9 @@ variable "allow_at_to_cos" {
   description = "Set rule for Activity Tracker to COS, default is true"
   default     = true
 }
+
+variable "allow_is_to_cos" {
+  type        = bool
+  description = "Set rule for IS to COS, default is true"
```
Comment (vburckhardt) [inline examples/fscloud/variables.tf:75]: I would just add a few words to explain what IS stands for here

**Review by vburckhardt — CHANGES_REQUESTED** [review]

File: examples/fscloud/variables.tf (hunk 1 above)
Comment (Ak-sky) [reply examples/fscloud/variables.tf]: Done.

**Review by vburckhardt — DISMISSED** [review]
lgtm

**Review by vburckhardt — APPROVED** [review]

**Pull request merged**